zerver/webhooks/jira/fixtures/comment_created_no_issue_details.json

```diff
@@ -16,7 +16,7 @@
       },
       "displayName": "Hemanth V. Alluri",
       "active": true,
-      "timeZone": "Asia/Calcutta",
+      "timeZone": "Asia/Calcutta"
```

Comment:
Jira isn’t really going to send us invalid JSON…right?

Reply:
@Hypro999 can you check this one -- is it possible that you hand-edited the payload?

Reply:
I do recall hand editing the payloads for Jira (I don't remember the exact details) but not this one. This fixture seems to have been added in commit 6df86dab3ef442b3e153a69f8133c7911b519d14.

Reply:
I see an valid JSON example at https://community.atlassian.com/t5/Jira-questions/July-5-2019-Change-to-Webhook-User-Shape/qaq-p/1127487. Did you perhaps hand-delete the `"accountType": "atlassian"` field without deleting the trailing comma?

Reply:
Seems very plausible.

Reply:
Merged under the assumption that this was a hand-editing error; if it wasn't, we'll find out when this reaches production, but it seems unlikely.